### PR TITLE
Fix form buttons

### PR DESCRIPTION
**Pull request by jeremyvergnas**
Fix form remove button

### CODE REVIEW

**Comment by doktordirk** [comment]
i still get
`aurelia-logging-console.js:54 ERROR [aurelia-form] an element key propery cannot be null or undefined`
though only once now since one button vanished.

File: src/scripts/page/form/demo.js [hunk 1]
```diff
@@ -50,15 +50,31 @@ function productForm() {
     ]
   };
 
-  let removeTag = (tag, index, $event) => {
+  let removeButton = {
```
Comment (bas080) [inline src/scripts/page/form/demo.js:53]: @jeremyvergnas, what is the type of the `removeButton` schema-element?

File: src/scripts/page/form/demo.js [hunk 2]
```diff
@@ -50,15 +50,31 @@ function productForm() {
...
+      label     : 'remove tag',
+      action    : ($event) => {removeTag(model.tags.length - 1, $event);},
+  attributes: {
```
Comment (bas080) [inline src/scripts/page/form/demo.js:56]: Something happened with the indentation

File: src/scripts/page/form/demo.js [hunk 3]
```diff
@@ -50,15 +50,31 @@ function productForm() {
...
+  class: 'btn btn-warning'
+  }
+};
+
+  let removeTag = (index, $event) => {
     model.tags.splice(index, 1);
+
+    if (!model.tags.length) {
+      return schema[2].actions.splice(1, 1);
```
Comment (bas080) [inline src/scripts/page/form/demo.js:65]: The goal is to remove the tags. Why are you splicing actions?

**Comment by jeremyvergnas** [comment]
This PR can be closed. It has been fixed in `aurelia-form`
https://github.com/SpoonX/aurelia-form/pull/56

Just need to be merged and then we can update swan-example-client with the latest version of `aurelia-form`

**Pull request closed without merging**